client/components/jetpack/connection-health/index.tsx

```diff
@@ -45,6 +56,75 @@ export const JetpackConnectionHealthBanner = ( { siteId }: Props ) => {
 		);
 	}
 
+	if ( errorType === FATAL_ERROR ) {
+		return (
+			<ErrorNotice
+				errorType={ errorType }
+				errorText={ translate(
+					"Jetpack is unable to communicate with your site due to a critical error that has occurred there. Please check your site admin's email inbox for instructions."
+				) }
+				noticeActionHref={ localizeUrl(
+					'https://wordpress.com/support/why-is-my-site-down/#theres-a-critical-error-on-your-site'
+				) }
+				noticeActionText={ translate( 'Learn how to fix' ) }
+			/>
+		);
+	}
+
+	if ( [ USER_TOKEN_ERROR, BLOG_TOKEN_ERROR ].includes( errorType ) ) {
+		return (
+			<ErrorNotice
+				errorType={ errorType }
+				errorText={ translate(
+					'Jetpack is unable to communicate with your site due to a token error. Please reconnect Jetpack.'
+				) }
+				noticeActionHref={ localizeUrl(
+					'https://wordpress.com/support/why-is-my-site-down/#theres-an-issue-with-your-sites-jetpack-connection'
+				) }
+				noticeActionText={ translate( 'Learn how to fix' ) }
+			/>
+		);
+	}
+
+	if ( errorType === HTTP_ERROR ) {
+		return (
+			<ErrorNotice
+				errorType={ errorType }
+				errorText={ translate(
+					'Jetpack is unable to communicate with your site due to a HTTP connection error. Please ensure that your site serves requests.'
+				) }
+				noticeActionHref={ localizeUrl( 'https://wordpress.com/support/why-is-my-site-down/' ) }
+				noticeActionText={ translate( 'Learn how to fix' ) }
+			/>
+		);
+	}
+
+	if ( errorType === PLUGIN_ERROR ) {
+		return (
+			<ErrorNotice
+				errorType={ errorType }
+				errorText={ translate(
+					'Jetpack is unable to communicate with your site due to an inactive Jetpack plugin. Please ensure that Jetpack plugin is active and reconnect Jetpack.'
```

Comment:
```suggestion
					'We are unable to communicate with your site because Jetpack is deactivated.'
```

It's a little confusing to say that 'Jetpack is unable to communicate with your site because it's deactivated...'

Reply:
Jetpack can mean different things, should we say `[...] because Jetpack plugin is deactivated`?

Reply:
@wojtekn Sure, I updated my suggestion.

Reply:
Let's say `We can't communicate…` instead of `We are unable to communicate…`